app/models/feedback.rb

```diff
@@ -51,6 +51,17 @@ def email_and_send
     send_report
   end
 
+  def rollout_string
+    string = ""
+    # ES UPGRADE TRANSITION #
+    # Remove ES version logic, but leave this method for future rollout use
+    string << if $rollout.active?(:use_new_search) || User.current_user.present? && $rollout.active?(:use_new_search, User.current_user)
```

Comment:
This can be `if use_new_search?`, inherited from `app/models/application_record.rb`.

Reply:
Sadly, it wasn't able to access the method when I tried that.

Reply:
I edited the file here so the tests can run, in case it was just my webdev being funny, which is always a possibility.